### PR TITLE
fix: ledger wallet signature

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
Ledger wallet seems to be subtracting 27 in `v` value during signature https://github.com/LedgerHQ/ledgerjs/issues/466 which leads to an invalid signature. 

# How
Add 27 in the `v` value. Thanks to @karmacoma-eth 💯 

# Test Plan
Test drop/claim works with ledger wallet

### CODE REVIEW

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/utilities.ts [hunk 1]
```diff
@@ -689,3 +689,17 @@ export function isIOS(): boolean {
...
   return isAndroid() || isIOS();
 }
+
+// TODO: https://github.com/LedgerHQ/ledgerjs/issues/466
+export const ledgerWalletHack = (signature?: string) => {
+  if (signature) {
+    const lastByteOfSignature = signature.slice(-2);
+    if (lastByteOfSignature === "00" || lastByteOfSignature === "01") {
+      const temp = parseInt(lastByteOfSignature, 16) + parseInt("1b", 16);
+      const newSignature = lastByteOfSignature + temp.toString(16);
```
Comment (0xkarmacoma) [inline packages/app/utilities.ts]:
```suggestion
      const newSignature = signature?.slice(0, -2) + temp.toString(16);
```

File: packages/app/utilities.ts [hunk 2]
```diff
@@ -689,3 +689,17 @@ export function isIOS(): boolean {
 export function isMobileWeb(): boolean {
   return isAndroid() || isIOS();
 }
+
+// TODO: https://github.com/LedgerHQ/ledgerjs/issues/466
+export const ledgerWalletHack = (signature?: string) => {
+  if (signature) {
+    const lastByteOfSignature = signature.slice(-2);
+    if (lastByteOfSignature === "00" || lastByteOfSignature === "01") {
+      const temp = parseInt(lastByteOfSignature, 16) + parseInt("1b", 16);
```
Comment (0xkarmacoma) [inline packages/app/utilities.ts]:
```suggestion
      const temp = parseInt(lastByteOfSignature, 16) + 27;
```

Comment (intergalacticspacehighway) [reply packages/app/utilities.ts]: optimization instincts. I like it 😂

**Pull request merged**